diff --git a/sesame-firmware-main.ino b/sesame-firmware-main.ino
index 92a1584..d669765 100644
--- a/sesame-firmware-main.ino
+++ b/sesame-firmware-main.ino
@@ -31,14 +31,14 @@
 //#define I2C_SDA 8
 //#define I2C_SCL 9
 
 // I2C Pins for Distro Board V1
-//#define I2C_SDA 21
-//#define I2C_SCL 22
+#define I2C_SDA 21
+#define I2C_SCL 22
 
 // I2C Pins for S2 Mini Board
-#define I2C_SDA 33
-#define I2C_SCL 35
+//#define I2C_SDA 33
+//#define I2C_SCL 35
 
 
 // DNS Server for Captive Portal
 DNSServer dnsServer;
@@ -103,12 +103,12 @@ Servo servos[8];
 // Sesame Distro Board V2 Pinout (Legacy)
 //const int servoPins[8] = {4, 5, 6, 7, 15, 16, 17, 18};
 
 // Sesame Distro Board V1 Pinout (Legacy)
-//const int servoPins[8] = {15, 2, 23, 19, 4, 16, 17, 18};
+const int servoPins[8] = {15, 2, 23, 19, 4, 16, 17, 18};
 
 // Lolin S2 Mini Pinout
-const int servoPins[8] = {1, 2, 4, 6, 8, 10, 13, 14};
+//const int servoPins[8] = {1, 2, 4, 6, 8, 10, 13, 14};
 
 // Subtrim values for each servo (offset in degrees)
 int8_t servoSubtrim[8] = {0, 0, 0, 0, 0, 0, 0, 0};
 
